### ISSUE TITLE
Ranking Loss

### ISSUE THREAD

**Issue by CosimoRulli** [thunlp/OpenMatch#10]
Hi developers, 
Looking at the code, I see you use the [MarginRankingLoss](https://pytorch.org/docs/stable/generated/torch.nn.MarginRankingLoss.html), while [Nogueira et al.](https://arxiv.org/pdf/1901.04085.pdf) suggest to adopt the cross-entropy loss. Is there a specific reason for this choice?

**Comment by zkt12** [comment]
Hi,

Our OpenMatch supports MarginRankingLoss and Cross-Entropy Loss, you can set the task as 'classification' for Cross-Entropy Loss, or 'ranking' for MarginRankingLoss :)

Kaitao

**Comment by CosimoRulli** [comment]
Thank you for your answer, but I did not explain myself clearly.  I was wondering if the ranking loss has some advantages w.r.t. the Cross-entropy. Which should I choose to maximize the performance of BERT?

**Comment by zkt12** [comment]
We found Cross-Entropy Loss is more stable for BERT training, especially in the scenarios when query-doc labels are not sufficient.
